README.md

```diff
@@ -81,14 +79,89 @@ project/
 
 .title {
   @include font-size(4rem);
-  // or
-  @include responsive-font-size(64px);
-  // or
-  @include rfs(64);
+
+  // The font-size mixin is a shorthand which calls
+  // @include rfs(4rem, font-size);
+
+  // Other shorthand mixins that are available are:
+  // @include padding(4rem);
+  // @include padding-top(4rem);
+  // @include padding-right(4rem);
+  // @include padding-bottom(4rem);
+  // @include padding-left(4rem);
+  // @include margin(4rem);
+  // @include margin-top(4rem);
+  // @include margin-right(4rem);
+  // @include margin-bottom(4rem);
+  // @include margin-left(4rem);
+
+  // For properties which do not have a shorthand, the property can be passed:
+  // @include rfs(4rem, border-radius);
+
+  // Whenever a value contains a space, it should be escaped with `#{}`:
+  // @include rfs(0 0 4rem red #{","} 0 0 5rem blue, box-shadow);
+
+  // Custom properties (css variables):
+  // @include rfs(4rem, --border-radius);
+}
+```
+
+If you're using Webpack, you can simplify the `@import` using the `~` prefix:
+
+```scss
+@import "~rfs/scss";
+```
+
+#### Generated css
+
+```css
+.title {
+  font-size: calc(1.525rem + 3.3vw);
+}
+
+@media (max-width: 1200px) {
```

Comment:
Should be `min-width` ? Other "Generated css" examples are same too.

Reply:
Thanks!